app/utility/reverse-invoice-number.js

```diff
@@ -0,0 +1,7 @@
+const reverseInvoiceNumber = (invoiceNumber) => {

```

Comment:
Can we get rid of `reverse-engineer-invoice-number` file sitting about?

This file needs tested -- the `reverse-engineer-invoice-number` might have some tests which cn be used here